README.md

```diff
@@ -11,13 +11,29 @@ A component library created for the cataloging components for [The OpenShift Web
 $ npm install
 $ bower install
 
-# start the server
+# build the library
+$ npm run build
+```
+
+## Showcase Application
+
+There is a showcase application used to help develop and view the existing components in the library.
+
+```
+# run the server
 $ npm run start
 ```
 
-Go to [https://localhost:9000](https://localhost:9000) in your browser.
+Go to [https://localhost:9001](https://localhost:9001) in your browser. (though it should auto-launch)
+This will watch for asset changes.
+
+You may need to patch oc to allow for port 9001:
+```
+$ oc login -u system:admin
+$ oc patch oauthclient/openshift-web-console -p '{"redirectURIs":["https://localhost:9001/"]}'
+```
 
->Warning: Make sure you're using the latest version of Node.js and NPM
+You may also need to update your service catalog project's deployment to update teh apiserver YAML to allow for port 9001 if it is configured for port 9000 only
```

Comment:
Yeah you can take this warning out, I already updated the template to take the port off so it will work for any port.